src/main/kotlin/pl/edu/uj/ii/ksi/mordor/services/upload/session/FileUploadSessionService.kt

```diff
@@ -0,0 +1,15 @@
+package pl.edu.uj.ii.ksi.mordor.services.upload.session
+
+import org.springframework.stereotype.Service
+
+@Service
+class FileUploadSessionService {
+    // TODO: Implementation
+    fun accept(uploadSession: FileUploadSession) {}
+
+    fun reject(uploadSession: FileUploadSession) {}
+
+    fun getAllSessions(): List<FileUploadSession> {
```

Comment:
make it accept a Pageable and return a Page - this will help with pagination in view.